subworkflows/local/annotation.nf

```diff
@@ -0,0 +1,93 @@
+/*
+    Run annotation tools
+*/
+
+include { PROKKA                         } from '../../modules/nf-core/prokka/main'
+include { PRODIGAL                       } from '../../modules/nf-core/prodigal/main'
+include { PYRODIGAL                      } from '../../modules/nf-core/pyrodigal/main'
+include { BAKTA_BAKTADBDOWNLOAD          } from '../../modules/nf-core/bakta/baktadbdownload/main'
+include { BAKTA_BAKTA                    } from '../../modules/nf-core/bakta/bakta/main'
+include { GUNZIP as GUNZIP_PRODIGAL_FNA  } from '../../modules/nf-core/gunzip/main'
+include { GUNZIP as GUNZIP_PRODIGAL_FAA  } from '../../modules/nf-core/gunzip/main'
+include { GUNZIP as GUNZIP_PRODIGAL_GBK  } from '../../modules/nf-core/gunzip/main'
+include { GUNZIP as GUNZIP_PYRODIGAL_FNA } from '../../modules/nf-core/gunzip/main'
+include { GUNZIP as GUNZIP_PYRODIGAL_FAA } from '../../modules/nf-core/gunzip/main'
+include { GUNZIP as GUNZIP_PYRODIGAL_GBK } from '../../modules/nf-core/gunzip/main'
+
+workflow ANNOTATION {
+    take:
+    fasta // tuple val(meta), path(contigs)
+
+    main:
+    ch_versions      = Channel.empty()
+    ch_multiqc_files = Channel.empty()
+
+        if ( params.annotation_tool == "pyrodigal" || ( params.annotation_tool == "prodigal" && params.run_bgc_screening == true && !params.bgc_skip_antismash ) ) { // Need to use pyrodigal for antiSMASH because prodigal GBK annotation format is incompatible with antiSMASH.
+
+            if ( params.annotation_tool == "prodigal" && params.run_bgc_screening == true && !params.bgc_skip_antismash ) {
```

Comment:
I sort feel this is overly complicated and wold be better solved with just blocking users requesting antiSMASH and prodigal at the same time (even if _theoretically_ it does the same thing)

Reply:
The warn would probably get lost if using ANSI_LOGGING on the console